site/content/blog/2020-05-02-a-growing-community.md

```diff
@@ -0,0 +1,42 @@
+---
+title: A growing community
+description: The Svelte community keeps on growing. Here's a summary of what's been happening over the past year.
+author: Johan Ronsse
+authorURL: https://twitter.com/wolfr_2_
+---
+
+It's been a year since the release of Svelte 3, and over the year, a lot has happened in terms of community. In this post, I am collecting some highlights of the past year, from a community perspective.
+
+This graph is only one slice of the pie, but here is the interest in Svelte as measured by Github stars over time.
+
+<img src="/blog/graph.png" style="max-width: 100%;">
+
+If you like Svelte and want to support it - give it a star [here](https://github.com/sveltejs/svelte).
+
+### General
+
+* Right after the last post on this website, Rich's talk [Rethinking Reactivity](https://www.youtube.com/watch?v=AdNJ3fydeao) got a lot of people excited about Svelte. He followed this talk up with [The return of write less, do more](https://www.youtube.com/watch?v=BzX4aTRPzno).
+* Near the end of last year, Svelte was a rising star in 2019's [State of JS](https://2019.stateofjs.com/front-end-frameworks/). It also received the "Prediction Award" for best upcoming technology.
+* In April, Svelte was featured in [Wired Magazine](https://www.wired.com/story/javascript-framework-puts-web-pages-diet/).
+* Svelte.js was talked about a lot in the overall development community. Ryan Atkinson's post [Why Svelte](https://github.com/feltcoop/why-svelte) became #1 on Hacker News.
```

Comment:
This is the only time in the post you're referring to is as 'Svelte.js', and this should probably be changed to 'Svelte', which is the preferred name anyway.